charts/memgraph/values.yaml

```diff
@@ -114,6 +114,8 @@ memgraphConfig:
   - "--storage-enable-schema-metadata=true"
   - "--storage-automatic-label-index-creation-enabled=true"
   - "--storage-automatic-edge-type-index-creation-enabled=true"
+  # for efficiently loading edges by ID
+  - "--storage-enable-edges-metadata=true"
```

Comment:
would it make sense to move it together with `- "--storage-enable-schema-metadata=true"` ?